src/olympia/blocklist/admin.py

```diff
@@ -387,15 +400,16 @@ def render_change_form(self, request, context, add=False, change=False,
 
     def save_model(self, request, obj, form, change):
         if change and self.is_pending_signoff(obj):
-            is_signoff = '_signoff' in request.POST
+            is_approve = '_signoff' in request.POST
```

Comment:
You're calling this `is_approve` because that is a clearer name, so should the `POST` param also be renamed from `_signoff` to `_approve`? I realize that might make for a bigger patch, but would it be worth doing in a follow-up?

Reply:
I was just being lazy https://github.com/mozilla/addons-server/pull/13622/commits/91e55bf467f9aa70128590fd5f99c65dd28f6f15